src/site/content/en/blog/mouse-acceleration/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Boost your FPS games on the web
```

Comment:
```suggestion
title: Disable mouse acceleration to provide a better FPS gaming experience
```
Continuing the convo from the Google Doc (thanks for avoiding `headshot`), when in doubt I look for a title that accurately describes the feature we're talking about. `Boost your FPS games on the web` doesn't give me much of a clue about what this page is actually talking about. If game developers search for this use case, I imagine they'll use `disable mouse acceleration` because the FPS gaming community in general probably knows about this tip